molecule/command/base.py

```diff
@@ -216,6 +216,7 @@ def click_group_ex():
     # yellow : special commands, like full test sequence, or login
     return click.group(
         cls=HelpColorsGroup,
+        context_settings=dict(max_content_width=9999),
```

Comment:
Would it be smart to put a comment in the code explaining why you've done this? Even if it's just `# Check commit msg on this one`?

Reply:
Usually blame can be used but in this case adding a commend makes sense, especially as the number seems weird. I will let the user engage with Pallets people around why logic values like None, 0 or -1 do not work..